interface/forms/note/new.php

```diff
@@ -83,9 +83,9 @@
 
                             <td>
                                 <span class="text"><?php echo xlt('Date'); ?></span>
-                                <input class="datepicker form-control" type='text' size='10' class='datepicker' name='date_of_signature' id='date_of_signature'
-                                       value='<?php echo attr(date('Y-m-d', time())); ?>'
-                                       title='<?php echo xla('yyyy-mm-dd'); ?>' />
+                                <input class="datepicker form-control" type='text' size='16' class='datepicker' name='date_of_signature' id='date_of_signature'
+                                       value='<?php echo attr(oeFormatDateTime(date('Y-m-d H:i', time()), $GLOBALS['time_display_format'])); ?>'
```

Comment:
Note if follow oeFormatDateTime to oeFormatTime, no need to set the 2nd parameter (if defaults to 'global' which then uses $GLOBALS['time_display_format'])

Reply:
note that when formatInput is used the entry needs to be standardized at some point before going into the daatabase (via the DateTimeToYYYYMMDDHHMMSS() function ).
This form also has a discrepancy in it's view.php for this date widget.

Reply:
So 2 things happen for internat to work on the datetimes.
A standard datetime is converted to the localize format via oeFormatDateTime in the datepicker
When saved or used in scripts it is converted back to the standard datetime via the DateTimeToYYYYMMDDHHMMSS function
(rec checking out DateTimeToYYYYMMDDHHMMSS use in code to get an idea how it works)